### ISSUE TITLE
Show all personal email reports, goals, alerts of all sites at once, not only the ones of the current selected site

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#7076]
When opening Custom Alerts Management we show all configured alerts of all websites. The name of the website an alert belongs to is displayed in a column.

On the contrary eg. in Email Reports we display only configured reports of the current selected site. To see all configured reports one has to switch through all websites separately which can take a long time. As there usually won't be many email reports we should display all of them at once similar to Custom Alerts.

Similar behaviour for Goals with one difference: There might be many Goals. Here we would need paging or a different solution. Or we would just leave it like it is currently. TBD

At least Email Reports and Custom Alerts should work the same way meaning displaying all entries of all websites by default. Still display a website selector so people can filter per website. Selecting "all websites" in site selector shows all entries of all websites again.

When changing this also the Email Report creation might need to be changed for consistency. In Custom Alerts one can select the website in the "Add new custom alert" form whereas this is not the case in "Email reports". Not saying one solution is better than the other only that we need to make sure they work the same way.

Re priority: Probably rather mid term than short term?

**Comment by mattab** [comment]
:+1:  with all

> At least Email Reports and Custom Alerts should work the same way meaning displaying all entries of all websites by default. Still display a website selector so people can filter per website. 
:+1: 
let's do it consistent for Email Reports + Custom Alerts first 
(and  do Goals later since it would need paging for large instances.)

(if we lower scope and don't do it for Goals maybe it could be done in Short term)

**Comment by tsteur** [comment]
Moving to short term and created a new issue for goals as suggested

**Comment by mattab** [comment]
I guess it's related to this request:  Superadmin is not able to list all segments #6766  as once this is done, we could also have the same UI pattern for segments (listing all across websites) as well as alerts and scheduled reports

**Comment by tassoman** [comment]
I was wondering about it in the forums: http://forum.piwik.org/t/scheduled-email-reports-super-administration/17657

**Comment by MatomoForumNotifications** [comment]
This issue has been mentioned on **Matomo forums**. There might be relevant details there:

https://forum.matomo.org/t/superadmin-can-not-manage-all-e-mail-reports/45104/2

**Comment by MatomoForumNotifications** [comment]
This issue has been mentioned on **Matomo forums**. There might be relevant details there:

https://forum.matomo.org/t/comment-visualiser-lensemble-des-rapports-envoyes-par-e-mail-sur-matomo/53025/2

**Comment by tsteur** [comment]
Refs https://github.com/matomo-org/matomo/issues/22343 (Add a Manage Segments section #22343)

**Comment by PeteTrombone** [comment]
That's a very good idea. We have a Matomo instance with over 300 websites (SiteId's). It's very difficult to find out where which of the functions mentioned are used.
We urgently need something like that.
